core/ROB.cpp

```diff
@@ -73,19 +73,9 @@ namespace olympia
     {
         for (auto & i : *in_reorder_buffer_write_.pullData())
         {
-            if (!i->isUOp())
```

Comment:
I changed this code in my most recent PR so you may have conflicts whenever you sync your fork.